docs/en/about-appium/platform-support.md

```diff
@@ -2,67 +2,101 @@
 
 Please refer to these driver docs for setup instructions.
 
-* Versions: 4.3 and up
-  * Versions 4.3 and up are supported via Appium's [UiAutomator and UiAutomator2](http://developer.android.com/tools/testing-support-library/index.html#UIAutomator)
-      libraries. UiAutomator is the default driver.
-* Devices: Android emulators and real Android devices
-* Native app support: Yes
-* Mobile web support: Yes. Automation
+- Versions: 4.3 and up
+  - Versions 4.3 and up are supported via Appium's [UiAutomator and UiAutomator2](http://developer.android.com/tools/testing-support-library/index.html#UIAutomator)
+    libraries. UiAutomator is the default driver.
+- Devices: Android emulators and real Android devices
+- Native app support: Yes
+- Mobile web support: Yes. Automation
   is effected using a bundled [Chromedriver](http://chromedriver.chromium.org)
   server as a proxy. With 4.3, automation works on official Chrome
   browser or Chromium only. With 4.4+, automation also works on the built-in
   "Browser" app. Chrome/Chromium/Browser must already be installed on the
   device under test. See the [mobile web doc](/docs/en/writing-running-appium/web/mobile-web.md) for instructions.
-* Hybrid support: Yes. See the [hybrid doc](/docs/en/writing-running-appium/web/hybrid.md) for instructions.
-  * With default Appium automation backend: versions 4.4 and up
-* Support for automating multiple apps in one session: Yes
-* Support for automating multiple devices simultaneously: Yes,
+- Hybrid support: Yes. See the [hybrid doc](/docs/en/writing-running-appium/web/hybrid.md) for instructions.
+  - With default Appium automation backend: versions 4.4 and up
+- Support for automating multiple apps in one session: Yes
+- Support for automating multiple devices simultaneously: Yes,
   though Appium must be started using different ports for the server
-   parameters `--port`, `--bootstrap-port` and/or
+  parameters `--port`, `--bootstrap-port` and/or
   `--chromedriver-port`. See the [server args doc](/docs/en/writing-running-appium/server-args.md) for more
   information on these parameters.
-* Support for automating vendor-provided or third-party apps: Yes
-* Support for automating custom, non-standard UI controls: No
+- Support for automating vendor-provided or third-party apps: Yes
+- Support for automating custom, non-standard UI controls: No
+
+### macOS Support
+
+macOS automation is supported with below drivers.
+
+- The [Mac2Driver](/docs/en/drivers/mac2.md) is for macOS 10.15 or later
+- The (deprecated) [MacDriver](/docs/en/drivers/mac.md) is for lower macOS versions
 
 ### Windows Desktop Support
 
-See the [Windows Driver](/docs/en/drivers/windows.md) doc for details
+Windows automation is supported with below drivers.
+
+- The [WinAppDriver](/docs/en/drivers/windows.md)
+- The [geckodriver](/docs/en/drivers/gecko.md) for Firefox and [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView)
+
+## Vendors/Community based drivers
```

Comment:
Maybe just a list instead of multiple subheaders here?

Reply:
yes. probably drivers will increase, so let me update after this merge.